src/app/pages/test-report/test-report.analytics.effects.ts

```diff
@@ -1128,7 +1128,7 @@ export class TestReportAnalyticsEffects {
     }),
   ));
 
-  // @TODO - MES-6879 - enable once component migrated
+  // @TODO - MES-6879 - enable once component migrated | Cat B Dependency
```

Comment:
This story is marked as done. Did we say Cat X+E uses this and it needs a new story

Reply:
Yes i believe that is correct